index.rst

```diff
@@ -3,11 +3,6 @@
 Symfony Documentation
 =====================
 
-.. toctree::
-    :hidden:
-
-    changelog
```

Comment:
Yeah it works. But it excludes the changelog from all the builds, so `changelog.html` would not be generated. I don't know if thats an issue for you guys or not.